apis/v1alpha1/httproute_types.go

```diff
@@ -23,6 +23,11 @@ import (
 type HTTPRouteSpec struct {
 	// Hosts is a list of Host definitions.
 	Hosts []HTTPRouteHost `json:"hosts,omitempty"`
+
+	// Gateways defines which Gateways can use this Route.
+	// +kubebuilder:default={allow: "SameNamespace"}
+	// +optional
+	Gateways *RouteGateways `json:"gateways,omitempty"`
```

Comment:
Since this field is being defaulted, you can remove `// +optional`.

Reply:
Updated, thanks!